layers/meta-balena-raspberrypi/recipes-bsp/bootfiles/rpi-config_git.bbappend

```diff
@@ -9,6 +9,8 @@ do_deploy:append() {
     if ! ${@bb.utils.contains('DISTRO_FEATURES','osdev-image','true','false',d)}; then
         echo "avoid_warnings=1" >>${DEPLOYDIR}/bootfiles/config.txt
     fi
+    # Remaps audio to GPIO12/13 freeing GPIO40/41 for SPI0 EEPROM communication
+    echo "dtoverlay=audremap" >> ${DEPLOYDIR}/bootfiles/config.txt
```

Comment:
Do you know if the audio jack still works?

Reply:
Testing on the CM4 which has no default audio jacks - but you're right the RPI4 will probably not work. Anyway, I will re-work it not to default to audio remapping.